src/promptflow-evals/samples/EvaluatorUpload.ipynb

```diff
@@ -0,0 +1,497 @@
+   "id": "846babb1-59b5-4d38-bb3a-d6eebd39ebee",
+   "metadata": {},
+   "source": [
+    "## End to end demonstration of evaluator saving and uploading to Azure.\n",
+    "### Saving the standard evaluators to the flex format.\n",
+    "First we will create the promptflow client, which will be used to save the existing flows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b03e557-19bd-4a2a-ae3f-bbaf6846fb33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pf = PFClient()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3fcaf38-6caa-4c1b-ada5-479686232cd1",
+   "metadata": {},
+   "source": [
+    "We will use F1 score evaluator from the standard evaluator set and save it to local directory. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "66fae04d-f6d0-4cc3-b149-a6058158c797",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pf.flows.save(F1ScoreEvaluator, path='./f1_score')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03552c5a-3ecc-4154-a0bd-e3fe2831e323",
+   "metadata": {},
+   "source": [
+    "Let us inspect, what has been saved"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0a27602c-e2c8-49c2-8d00-1eb5a11e55a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('\\n'.join(os.listdir('f1_score')))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7dc4dbfe-faa0-44f9-a53e-310c946d91fe",
+   "metadata": {},
+   "source": [
+    "The file, defining entrypoint of our model is called flow.flex.yaml, let us display it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7c30c3e3-8113-4e0f-9210-b062e7354099",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(os.path.join('f1_score', 'flow.flex.yaml')) as fp:\n",
+    "    flex_definition = yaml.safe_load(fp)\n",
+    "print(f\"The evaluator entrypoint is {flex_definition['entry']}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d9bb9a12-8bdd-4679-9957-998f3c7ceb75",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pf = PFClient()\n",
+    "run = Run(\n",
+    "    flow='f1_score',\n",
+    "    data='data.jsonl',\n",
+    "    name=f'test_{uuid.uuid1()}'\n",
+    ")\n",
+    "run = pf.runs.create_or_update(\n",
+    "     run=run,\n",
+    ")\n",
+    "pf.stream(run)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e7ee69f-4db3-4f5f-874a-5dc95d2e2f7c",
+   "metadata": {},
+   "source": [
+    "<font color='red'>**Hack for standard evaluators.**</font> If we will try to load our evaluator directly, we will get an error, because we try to modify `__path__` variable, which will work inside the python package, however, it will fail if we will try to run evaluator as a standalone script. To fix it, we will remove this line from our code. **This code is not needed for custom user evaluators!**"
```

Comment:
so if we can remove  `__path__ = __import__("pkgutil").extend_path(__path__, __name__)`  from `__init__.py`, we don't need below workaround, correct?